pallets/bridge/src/migration.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2022 Centrifuge Foundation (centrifuge.io).
+// This file is part of Centrifuge Chain project.
+
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+use super::*;
+
+pub mod fix_pallet_account {
+	use cfg_primitives::AccountId;
+	#[cfg(feature = "try-runtime")]
+	use frame_support::ensure;
+	use frame_support::{log, weights::Weight};
+	use scale_info::prelude::format;
+	use sp_runtime::traits::{AccountIdConversion, Zero};
+	use sp_std::vec;
+
+	use super::*; // Not in prelude for try-runtime
+
+	const WRONG_INBOUND_ID: PalletId = PalletId(*b"cb/bridg");
+	const WRONG_OUTBOUND_ID: PalletId = cfg_types::ids::BRIDGE_PALLET_ID;
```

Comment:
We could delete the `BRIDGE_PALLET_ID` type and just reference manually the PalletId as you do above.

Reply:
yeah sounds good!